packages/validator/src/services/indices.ts

```diff
@@ -101,15 +125,39 @@ export class IndicesService {
     ApiError.assert(res, "Can not fetch state validators from beacon node");
 
     const newIndices = [];
+
+    const allValidatorStatuses = new MapDef<SimpleValidatorStatus, number>(() => 0);
+
     for (const validatorState of res.response.data) {
+      // Group all validators by status
+      const status = statusToSimpleStatusMapping(validatorState.status);
+      allValidatorStatuses.set(status, allValidatorStatuses.getOrDefault(status) + 1);
+
       const pubkeyHex = toHexString(validatorState.validator.pubkey);
       if (!this.pubkey2index.has(pubkeyHex)) {
-        this.logger.debug("Discovered validator", {pubkey: pubkeyHex, index: validatorState.index});
+        this.logger.info("Validator exists in beacon chain", {
```

Comment:
@maschad running this on goerli with quite a lot of validators and this inital log is quite noisy in my opinion, we are printing out `Validator exists in beacon chain` for each validator which seems bit redundant because these logs all happen after each other.

I think we should reconsider this and maybe only print out `Validator exists in beacon chain` once and then just list all the validators, with index, pubkey and ideally also fee recipient (if possible)

Reply:
Good point

Reply:
Good point, I've included that change in be5a7e4ae5f9e70c62a3b46d2639e35b3ccedd30